storage/Utils/Logger.cc

```diff
@@ -155,7 +132,24 @@ namespace storage
 	--count;
     }
 
-
     int Silencer::count = 0;
 
+
+    const string default_log_path()
```

Comment:
Please use either a plain `string` or a `const string &`. Just `const string` does not serve any purpose.